backend/src/test/api/endpoints/test_general.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from httpx import AsyncClient
```

Comment:
We probably need to add this to `backend/environment.yml`?

Reply:
When do I build the container, after or before this PR is merged?

Reply:
It's better to build the container before just to make sure it builds and has no dep conflicts etc

Reply:
Oh, and I probably don't have access to the docker hub for pushing the image.

Reply:
Tried to build the container, got an error :D I'll investigate

Reply:
I didn't manage to build the container, but building it fails on the `mwp_v1` branch as well. So probably something is weird with my docker setup. 

The new packages in this PR are used for testing only, so maybe we can merge without pushing a new docker image?

Reply:
Managed to fix my setup :) The build now runs fine! I still don't think I can push to the registry due to missing access, though.

Reply:
Yes, sorry forgot that, I pushed it a few days ago :)